pennylane/drawer/tape_text.py

```diff
@@ -206,7 +198,7 @@ def _add_cwire_measurement(m, layer_str, config):
     layer_str = _add_cwire_measurement_grouping_symbols(mcms, layer_str, config)
 
     mv_label = "MCM"
-    meas_label = measurement_label_map[m.return_type](mv_label)
+    meas_label = measurement_label_map[m._shortname](mv_label)  # pylint: disable=protected-access
```

Comment:
```suggestion
    meas_label = _measurement_label_map[type(m)](mv_label)_  # pylint: disable=protected-access
```
Maybe something like this together with the suggestion above?
The idea it that the class type itself provides as much information as the `return_type`, so where ever `return_type` was being used, we can replace it with checks on the class.